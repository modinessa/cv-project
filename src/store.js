import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
