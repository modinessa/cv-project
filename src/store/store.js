import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});