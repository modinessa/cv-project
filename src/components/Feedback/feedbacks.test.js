import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Feedback } from './index';

const TEST_USERS = [{
    name: 'Name', 
		photo: "https://photo.jpg",
    feedback: 'Loreusum. Integer tincidunt. Cras dapibus.',
		company: "A Game of Thrones",
		website: "https://somesite.com/"
	}];


describe('Feedback', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Feedback
					users={TEST_USERS}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})