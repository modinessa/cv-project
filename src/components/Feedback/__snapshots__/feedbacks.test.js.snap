// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback renders component 1`] = `
<section
  className="app-section"
>
  <h2>
    Feedbacks
  </h2>
  <div
    className="app-feedbacks"
  >
    <div
      className="app-feedbacks-user"
    >
      <div
        className="info"
      >
        <h3 />
        <p>
          Loreusum. Integer tincidunt. Cras dapibus.
        </p>
      </div>
      <div
        className="user-info"
      >
        <img
          alt="Name"
          className="app-photo user-photo"
          src="https://photo.jpg"
        />
        <div>
          A Game of Thrones,
        </div>
        <a
          href="https://somesite.com/"
        >
          somesite.com
        </a>
      </div>
    </div>
  </div>
</section>
`;
